taipy/gui/custom/_page.py

```diff
@@ -48,6 +51,8 @@ class ResourceHandler(ABC):
 
     rh_id: str = ""
 
+    data_layer_supported_types: t.Tuple[t.Type, ...] = (_TaipyData, pd.DataFrame, pd.Series)
```

Comment:
is `_TaipyData `necessary here ?

Reply:
To show that we want to deal with that. I would want it here.